moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -46,6 +50,7 @@ const validationSchema = yup.object().shape({
   tags: yup.array().optional(),
   completionDate: yup.date().nullable().optional(),
   description: yup.string(),
+  work_types: yup.array().of(yup.object()).min(1).required(),
```

Comment:
Cool! I wondered how the validation worked when I tested the form. 👍

Reply:
yup!